src/__tests__/device-test.ts

```diff
@@ -4,6 +4,7 @@ import {
     requestDeviceImei,
     internalNavigation,
     dismiss,
+    requestVibration,
```

Comment:
I'd name this `vibrate`

Reply:
Left it as requestVibration as some devices won't do anything and we won't receive a confirmation.